docs/README-AWS.md

```diff
@@ -398,70 +479,99 @@ inputs:
           es_datastream_name: "logs-generic-default"
 ```
 
-#### Notes
-`inputs.[].expand_event_list_from_field` can be defined as string with the value of a key in the JSON that contains a list of elements that must be sent as events instead of the encompassing JSON.
+**Notes:**
+
+`inputs.[].expand_event_list_from_field` can be defined as a string with the value of a key in the JSON that contains a list of elements that must be sent as events instead of the encompassing JSON.
+
+You should note that when relying on the [Routing support for AWS Service Logs](#routing-support-for-aws-services-logs), any value set for the `expand_event_list_from_field` configuration parameter will be ignored, because this will be automatically handled by the Elastic Serverless Forwarder.
```

Comment:
the internal link is now `[Routing AWS Service Logs](#routing-aws-service-logs)`